tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+>> Start next lesson
+
+==== Admonitions
+Admonitions are signal words used to catch the reader’s attention, such as “TIP”, “NOTE”, or “WARNING”. There are two ways to declare an admonition: inline and as a block.
+
+*Inline Admonitions*
+
+Start a new line with the signal word in all caps and a colon and write your admonition.
+
+.Example for an inline note
+[source, AsciiDoc]
+----
+NOTE: Advice on when to use which signal word is specified in ANSI Z535.6. 
+----
+
+*Block admonitions*
+
+. Start with the signal word in all caps enclosed in square brackets.
+. Insert the block delimiter.
+. Insert any AsciiDoc markup that you need.
+. End the block with a block delimiter.
+
+.Example for a note block.
+[source, AsciiDoc]
+------
+[NOTE] <1>
+==== <2>
+This is an example of an admonition block. <3>
+
+Unlike an admonition paragraph, it may contain any AsciiDoc content.
+The style can be any one of the admonition labels:
+
+* NOTE
+* TIP
+* WARNING
+* CAUTION
+* IMPORTANT
+==== <4>
+------
+
+If an admonition is valid for a whole section or a whole document and should appear before any content in the section, you can set the `beforeclauses` attribute like this:
+
+[source, AsciiDoc]
+------
+[IMPORTANT,beforeclauses=true]
+====
+This important notice applies to the entire document.
+====
+------
+
+[[exercise-2-7]]
+include::exercises.adoc[tag=exercise-2-7]
+
+Great progress so far! Let's look at code samples in the next lesson.
+>> Start next lesson
+
+==== Code Samples
+
+AsciiDoc supports code samples using the `[source]` attribute before a new block. If you specify the coding language, the sample will be highlighted correctly. Have a look at the example:
+
+[source, AsciiDoc]
+------
+.Sample Code <1>
+[source,ruby] <2>
+==== <3>
+puts "Hello, world." <4>
+%w{a b c}.each do |x|
+  puts x
+end 
+==== <5>
+------
+
+<1> Title for your code sample (optional)
+<2> `[source]` specifies that the block contains code. `ruby` is the coding language. The block will highlight keywords depending on the coding language.
+<3> Starting delimiter
+<4> Sample code
+<5> Ending delimiter
+
+[[exercise-2-8]]
+include::exercises.adoc[tag=exercise-2-8]
+
+We’re done with blocks - good job! The next lesson covers inline markup. 
+>> Start next lesson
+
+=== Inline Markup
+
+To annotate single words, you use inline markup. The markup encloses the word(s) and you do not need to begin a new line. Inline markup allows you to:
+
+* Emphasize text
+* Link to external resources with hyperlinks
+* Create internal cross-references
+* Create index entries
+* Create Bibliographic entries
+
+Let’s look at these features in the following lessons.
+>> Start next lesson
+
+==== Text markup
+
+The simplest form of inline markup is to emphasize text. 
+AsciiDoc allows you to:
+
+* Make words *bold* using asterisks 
+* _Italicise_ words with underscores 
+* Apply `monospacing` using backticks
+
+[source, AsciiDoc]
+---- 
+*bold*
+_italic_
+`monospace`
+----
```

Comment:
You could add `^superscript^` and `~subscript~`; technical writers will want those, and they should not need to resort to AsciiMath or LaTeX to get them.